org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_RunnableDevice.java

```diff
@@ -169,6 +169,7 @@ public boolean isActivated() {
 	public boolean setActivated(boolean activated) throws ScanningException {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "@Override" annotation above this method signature [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1161)